src/basis/Monad.ml

```diff
@@ -19,6 +19,9 @@ module type Notation = sig
   val (|>>) : 'a m -> ('a -> 'b m) -> 'b m
   val (@<<) : ('a -> 'b m) -> 'a m -> 'b m
   val (<&>) : 'a m -> 'b m -> ('a * 'b) m
+  val (>>=) : 'a m -> ('a -> 'b m) -> 'b m
+  val (<@>>) : ('a -> 'b) -> 'a m -> 'b m
```

Comment:
Btw I don't know if it helps for me to mention this, but these two operators are already present a few lines above --- with different names.

Reply:
oh, good point. i was mostly just matching ASCII art to get things to type check last week, but that's a redundancy that needn't exist. i'll clean it up.